nvflare/app_opt/flower/proto/grpcadapter.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package flower.proto;
```

Comment:
Yes, but we need to change the package name from flwr.proto to flower.proto, otherwise protobuf will complain about namespace conflict when both Flare and Flower are installed.

Reply:
Added comments to proto definition.